input/fsh/Clinical-Response-01-Baseline.fsh

```diff
@@ -0,0 +1,234 @@
+    * answerOption[+].valueString = "Invasive lobular carcinoma"
+    * answerOption[+].valueString = "Other"
+    * answerOption[+].valueString = "Unknown"
+    * required = true
+    * repeats = true
+
+  * item[+]
+    * linkId = "MUTBC-01"
+    * type = #choice
+    * text = "Indicate if the patient carries a genetic mutation predisposing breast cancer"
+    * answerOption[+].valueString = "No mutation"
+    * answerOption[+].valueString = "BRCA 1"
+    * answerOption[+].valueString = "BRCA 2"
+    * answerOption[+].valueString = "Other mutation"
+    * answerOption[+].valueString = "Not tested"
+    * required = true
+
+  * item[+]
+    * linkId = "GRADEINV-01"
+    * type = #choice
+    * text = "Indicate the grade of the invasive component of tumor"
+    * answerOption[+].valueString = "Grade 1"
+    * answerOption[+].valueString = "Grade 2"
+    * answerOption[+].valueString = "Grade 3"
+    * answerOption[+].valueString = "Not reported"
+    * required = true
+
+  * item[+]
+    * linkId = "GRADEDCIS-01"
+    * type = #choice
+    * text = "Indicate the tumor grade of DCIS component of tumor"
+    * answerOption[+].valueString = "Low"
+    * answerOption[+].valueString = "Intermediate"
+    * answerOption[+].valueString = "High"
+    * answerOption[+].valueString = "Not reported"
+    * required = true
+
+  * item[+]
+    * linkId = "TNMCT_BREAST-01"
+    * type = #choice
+    * text = "Indicate the clinical tumor stage (per AJCC 5th - 7th Ed.)"
+    * answerOption[+].valueString = "cTX"
+    * answerOption[+].valueString = "cT0"
+    * answerOption[+].valueString = "cTis"
+    * answerOption[+].valueString = "cT1"
+    * answerOption[+].valueString = "cT2"
+    * answerOption[+].valueString = "cT3"
+    * answerOption[+].valueString = "cT4"
+    * answerOption[+].valueString = "Unknown"
+    * required = true   
+
+  * item[+]
+    * linkId = "TNMCN_BREAST-01"
+    * type = #choice
+    * text = "Indicate the clinical nodal stage (per AJCC 5th - 7th Ed.)"
+    * answerOption[+].valueString = "cNX"
+    * answerOption[+].valueString = "cN0"
+    * answerOption[+].valueString = "cN1"
+    * answerOption[+].valueString = "cN2"
+    * answerOption[+].valueString = "cN3"
+    * answerOption[+].valueString = "Unknown"
+    * required = true
+
+  * item[+]
+    * linkId = "TNMCM_BREAST-01"
+    * type = #choice
+    * text = "Indicate the clinical distant metastasis (per AJCC 5th - 7th Ed.)"
+    * answerOption[+].valueString = "cMX"
+    * answerOption[+].valueString = "cM0"
+    * answerOption[+].valueString = "cM1"
+    * answerOption[+].valueString = "Unknown"
+    * required = true
+
+  * item[+]
+    * linkId = "ERSTATUS-01"
+    * type = #choice
+    * text = "Indicate if the estrogen receptor status is positive"
+    * answerOption[+].valueString = "No"
+    * answerOption[+].valueString = "Yes"
+    * answerOption[+].valueString = "Not performed"
+    * answerOption[+].valueString = "Unkown"
```

Comment:
We re-use these answers in a couple of places, like the question below, so this should be a valueset. We can use the codes from https://www.hl7.org/fhir/valueset-example-yesnodontknow.html and add 'not performed' into it. But let's hold off on that until we solve the https://ichom.atlassian.net/browse/BCFI-33